src/arch/Sound/RageSoundDriver_WaveOut.h

```diff
@@ -28,11 +27,14 @@ class RageSoundDriver_WaveOut: public RageSoundDriver
 
 	HWAVEOUT m_hWaveOut;
 	HANDLE m_hSoundEvent;
-	WAVEHDR m_aBuffers[NUM_BUFFERS];
+	WAVEHDR m_aBuffers[32]; // Maximum of 32 output blocks (frame blocks) allowed.
 	int m_iSampleRate;
 	bool m_bShutdown;
 	int m_iLastCursorPos;
 	bool b_InitSuccess;
+	int NUM_CHUNKS; // TODO rename wo_num_blocks
```

Comment:
Can we fix all the namings in this PR? I don't see why they should be the way they currently are.